src/ape/api/query.py

```diff
@@ -12,6 +12,7 @@
     "AccountTransactionQuery",
     "ContractEventQuery",
```

Comment:
Was thinking this is the query type to use for querying events through the query manager

NOTE: We can make changes to this type